init.lua

```diff
@@ -67,11 +67,19 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 
 	for key, value in pairs(colors) do
 		if fields[key] then
-			player:set_nametag_attributes({
-            			color = colors[key]
-			})
+			player:set_nametag_attributes({color = colors[key]})
+			player:set_attribute("nametag_color", minetest.serialize(colors[key]))
+			minetest.chat_send_player(player:get_player_name(), "Your nametag color has been set to: " .. key)
 		end
-   	end
+	end
+end)
+
+minetest.register_on_joinplayer(function(player)
+	local color = minetest.deserialize(player:get_attribute("nametag_color"))
```

Comment:
`get_attribute` is deprecated. Please use `player:get_meta():set_string(`

Reply:
It's for saving compatibility with MT 0.4